src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
+	 * Enables logging in the service provider, if not already enabled.
+	 *
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 68858175c20032ebc8395d87f02bc6d6</sub>